package.json

```diff
@@ -19,9 +19,9 @@
     "docs-build": "gatsby build",
     "docs-deploy": "vercel && vercel alias $(pbpaste) mdxjs.com && vercel alias $(pbpaste) www.mdxjs.com",
     "format": "yarn lint --fix",
-    "lint": "eslint . --cache --ext js,jsx -f friendly",
+    "lint": "eslint . --cache --ext md,mdx,js,jsx -f friendly",
     "vercel-build": "yarn build && yarn docs-build",
-    "postinstall": "yarn-deduplicate || exit 0",
+    "postinstall": "yarn-deduplicate --strategy fewer && node scripts/patch || exit 0",
```

Comment:
:thinking: why are scripts being patched in a documentation change?

Reply:
See https://github.com/mdx-js/mdx/pull/1490#discussion_r600558337